src/main/resources/db/migration/V1__Create_db.sql

```diff
@@ -0,0 +1,46 @@
+create table DRIVER
+(
+    ID        int         not null primary key,
+    NAME      varchar(64) not null,
+    PASSPORT  varchar(10) not null,
+    LICENSE   varchar(10) not null,
+    PHONE_NUM varchar(11),
+    LOGIN     varchar(64) not null,
+    PASSWORD  varchar(64) not null
+);
+
+create table TICKET
+(
+    ID        int     not null primary key,
+    ID_CLIENT int     not null references DRIVER (ID),
+    IS_PAID   boolean not null,
+    COST      int     not null
+);
+
+create table ADMINISTRATOR
+(
+    ID       int         not null primary key,
+    NAME     varchar(64) not null,
+    LOGIN    varchar(64) not null,
+    PASSWORD varchar(64) not null
+);
+
+create table CAR
+(
+    ID     int         not null primary key,
+    MARQUE varchar(16),
+    MODEL  varchar(16),
+    NUMBER varchar(10) not null
+);
+
+create table OFFER
+(
+    ID         int         not null primary key,
```

Comment:
If we use surrogate key like ID, it's worth to create sequences or use type serial to autoincrement this value, we are definitely do not want to remember witch one ID we should set next to insert it into database.
Apply it everywhere please